app/lib/frontend/request_context.dart

```diff
@@ -14,7 +14,22 @@ RequestContext get requestContext =>
 
 /// Holds flags for request context.
 class RequestContext {
+  /// Whether the request is on a production host (vs. staging or dev)
+  final bool isProductionHost;
```

Comment:
What is this used for anymore?

Reply:
It is used to guard the cache updates in certain pages. Even if it is not used anymore, I'd rather remove it after next week.